addon/components/au-fieldset/legend.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@glimmer/component';
+
+export default class AuFieldsetLegend extends Component {
+  get skin() {
+    if (this.args.skin == '1') return 'au-u-h1';
+    if (this.args.skin == '2') return 'au-u-h2';
+    if (this.args.skin == '3') return 'au-u-h3';
+    if (this.args.skin == '4') return 'au-u-h4';
+    if (this.args.skin == '5') return 'au-u-h5';
+    if (this.args.skin == '6') return 'au-u-h6';
+    if (this.args.skin == 'functional') return 'au-u-h-functional';
```

Comment:
What's this for exactly?

Reply:
So you can change the font size of the legend when using this as a fieldset grouping multiple elements. I added an example for this.

Reply:
Yea, but specifically the `functional` version?

Reply:
That's a title size used in Kaleidos and webuniversum. I want to provide the same sizing options as a title here.

Reply:
Ok :+1: Hadn't encountered it before.

Reply:
Would it make sense to also add a `label` skin? The cases I've seen would just want it to look like a label instead of a title.

Reply:
The default size and styling is the same as a label or (au-u-h6) so not sure if we need it as a seperate option.